src/entry-client.js

```diff
@@ -16,9 +64,13 @@ export default function initializeClient(createApp, clientOpts) {
         middleware: () => Promise.resolve(),
         postMiddleware: () => Promise.resolve(),
         logger: console,
+        enablePerfMarks: false,
```

Comment:
I agree it was getting messy before (my bad, since the function was my idea in the first place) - letting the app determine the internal route name format seemed like overkill - I can't imagine a scenario at the moment that `Homepage->Product` is insufficient, so let's take the easy road to start